pgxn/hnsw/hnsw.c

```diff
@@ -580,6 +580,7 @@ l2_distance(PG_FUNCTION_ARGS)
 				 errmsg("different array dimensions %d and %d", a_dim, b_dim)));
 	}
 
+	#pragma clang loop vectorize(enable)
```

Comment:
It's better to make changes in this repo: https://github.com/neondatabase/pg_embedding

Reply:
Done